source/includes/_187-roles.md.erb

```diff
@@ -0,0 +1,7 @@
+# Roles
+
+The roles API allows admin users to manage roles or plugin roles, The plugin roles later used by plugin to authorize user for GoCD server.
+
+In order to create `Plugin Role` authorization configuration is required. [Click here](#authorization-configuration) know more about authorization configuration API
```

Comment:
In order to create a `plugin role`, [authorization configuration](#authorization-configuration) is required.